example/lib/screens/apple_pay_screen.dart

```diff
@@ -3,7 +3,6 @@ import 'dart:convert';
 import 'package:flutter/material.dart';
 import 'package:http/http.dart' as http;
 import 'package:flutter_stripe/flutter_stripe.dart';
-import 'package:stripe_example/widgets/loading_button.dart';
```

Comment:
In general I am not convinced of moving the example to it's own folder. If I look to the federated plugins of google they all put the example inside the main dart package. See for example https://github.com/flutter/plugins/tree/master/packages/url_launcher . I think it will be confusing where the example is for in this case.
- [ ] Revert moving the example out of the stripe package

Reply:
I would think it is easier to find like that

Reply:
Hey! @remonh87
Yes, for the Flutter official plugins, I think it makes a lot of sense to put the example folder inside the Dart package. But that is because the [`flutter/plugins`](https://github.com/flutter/plugins) is a mono-repo for multiple published plugins with different functionalities.

But if you notice the [`flutter/flutter`](https://github.com/flutter/flutter), they actually have the `examples` folder in the root level, containing multiple examples for the Flutter repository which in my opinion helps a lot since it's for folks to discover the `example` folder as well (I doubt a lot of people will go to the `stripe` folder unless they are interested in contributing or we mention it explicitly on the README.md).

I think putting the `example` in the `stripe` folder would be better if we plan to have a separate example for each of the folder: `stripe_android`, `stripe_ios`, and `stripe_platform_interface`. Are we planning to do that?
Because currently it's not obvious where the example is located in the codebase if I'm particularly just a user interested in using this package and reading the README.md of this repository.

Since technically this repository should contain one functionality: the Stripe SDK, I think it's much better to have the `example` in the root level, following the example of the [Stripe React Native SDK repository](https://github.com/stripe/stripe-react-native).

Another reason why I moved the `example` is because of https://github.com/flutter-stripe/flutter_stripe/runs/2647177553 in this PR was failing because of an error in the `example` folder due to the `.env.dart` missing in the plugin.. Running `flutter analyze` on the `stripe` folder would run the analyzer twice: one on the `example` folder, and then another in the current `stripe` package codebase.
Any suggestion on how to resolve this issue if you have a strong opinion to still keep this `example` folder in `stripe` level?

@jonasbark easier to find like that meaning in the `stripe` folder or in the root level?

Reply:
I'd definitely vote for it to be in the root folder. 
But no separate example per platform plugin

Reply:
Ah, I see. Thanks, @jonasbark !
Yeah, I'm also in strong favor of having the example in the root folder for better visibility as well, since I don't think we can also have a useful example per platform plugin?

Reply:
I understand the rationale my comment is more that we make a conscious decision to deviate. We should make sure that the example is correctly referred to in pub.dev

Reply:
definitely, we should make sure to set the example correctly on the `pub.dev` website as well.
Sorry @remonh87, I'm still quite unclear here, so are you still in favor of reverting the changes on keeping the `example` on `stripe` or would you be ok with moving this `example` on the root level here?

Would also love to hear your opinion since I know you have been working and changing a lot of the `example` code, want to make sure this change is ok with you! @jamesblasco

Reply:
I would also prefer to have it at the root level. 

And for pub.dev is better, right now the repo link is `https://github.com/flutter-stripe/flutter_stripe/tree/main/stripe` to allow the example to appear. With this change, we can have `https://github.com/flutter-stripe/flutter_stripe/`

Reply:
awesome, will resolve this conversation.